test/visibility/import/rename/renameChain.good

```diff
@@ -1,3 +1,4 @@
-renameChain.chpl:5: warning: identifier 'bar' is repeated
-note: Did you mean to rename 'foo' to 'baz'?
-renameChain.chpl:5: error: Bad identifier in rename, no known 'bar' in 'M'
+renameChain.chpl:4: In module 'User':
+renameChain.chpl:5: error: cannot rename symbol 'bar' as it is not defined in 'M'
+renameChain.chpl:5: warning: 'bar' is repeated
+renameChain.chpl:5: note: did you mean to rename 'foo' to 'baz'?
```

Comment:
Also a pre-existing condition (so also not a show-stopper), but it surprises me we complain about 'bar' being repeated after we've errored about not being able to rename it.  From previous examples, I'd need to do `import M.{bar, bar}` or `import M.{bar, bar as goo}` in order to get the repeated warning; but here I'm getting it from the result of a renaming even though that renaming itself is being flagged as illegal?  That seems confusing.  The "Did you mean to..." guess also seems like an unhelpful guess since we'd then warn about 'foo' being repeated (so we should either start permitting symbols to be imported under different names without warning, or not suggest the user do so if it would result in a repetition that we'd warn about?)